docs/rst/user_manual/configuration.rst

```diff
@@ -223,8 +254,59 @@ Apart from these values, the tag ``qos`` under each topic allows to configure th
         - ``false``
         - Topic with / without key
 
-The entry ``keyed`` determines whether the corresponding topic is `keyed <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/topic/typeSupport/typeSupport.html#data-types-with-a-key>`_
-or not. See the :term:`Topic` section for further information about the topic.
+    *   - Max Transmission Rate
+        - ``max-tx-rate``
+        - *float*
+        - *default value*
+        - Maximum sample transmission rate [Hz]
+
+    *   - Max Reception Rate
+        - ``max-rx-rate``
+        - *float*
+        - *default value*
+        - Maximum sample reception rate [Hz]
+
+    *   - Down-sampling
+        - ``downsampling``
+        - *integer*
+        - *default value*
+        - Down-sampling factor
+
+The entry ``keyed`` determines whether its corresponding topic is `keyed <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/topic/typeSupport/typeSupport.html#data-types-with-a-key>`_ or not.
+For further information on topics, please read the `Topic <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/topic/topic.html>`_ section.
+
+.. _user_manual_configuration_manual_topics:
+
+Topics
+------
+
+A subset of QoSs can be manually configured for a specific topic under the tag ``topics``.
+The tag ``topics`` has a required ``name`` tag that accepts wildcard characters.
+It also has three optional tags: a ``type`` tag that accepts wildcard characters, a ``qos`` tag with the QoSs that the user wants to manually configure, and a ``participants`` tag that lists the participants to which the configuration applies.
+If a ``qos`` is not manually configured, it will get its value by discovery; if there ``participants`` tag is empty or non-existent, the configuration will apply to all participants.
```

Comment:
```suggestion
If a ``qos`` is not manually configured, it will get its value by discovery; if the ``participants`` tag is empty or non-existent, the configuration will apply to all participants.
```